x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/cluster/ClusterStatsMonitoringDocTests.java

```diff
@@ -446,7 +446,7 @@ public void testToXContent() throws IOException {
                         + "\"ingest\":0"
                       + "},"
                       + "\"versions\":["
-                        + "\"6.0.0-alpha2\""
+                        + "\"7.0.0\""
```

Comment:
This string needs to be based on the version returned above, otherwise it will require updating when current is bumped to a new major.